spec/parsers/mp3_parser_spec.rb

```diff
@@ -71,6 +71,14 @@
     expect(prepped.pos).to eq(3145738)
   end
 
+  it 'returns only ID3V1 if some member of ID3V2 raise an encoding error' do
+    fpath = fixtures_dir + '/MP3/invalid_encoding_id3v2_artist.mp3'
```

Comment:
I didn't push this file because I don't have the permission to use it here as a fixture.

I will try to remove the content as I did in https://github.com/WeTransfer/format_parser/issues/155#issuecomment-675677078 for a zip file.